src/eyed3/utils/__init__.py

```diff
@@ -49,7 +49,12 @@ def guess_type(self, filename):
         if os.path.splitext(filename)[1] in ID3_MIME_TYPE_EXTENSIONS:
             return ID3_MIME_TYPE
         try:
-            return self.from_file(filename)
+            mtype = self.from_file(filename)
+            if str(mtype).startswith("cannot open"): #There are problems with path encoding in libmagic; Use workaround if can't read the file due to encoding
```

Comment:
I think this conditional should include `sys.platform == "windows"`, or whatever the right value is for Windows. I don't have a Windows system to test or see the platform value.